extension/app/src/lib/file.ts

```diff
@@ -0,0 +1,18 @@
+// These are the front-end helpers.
+
+function isMarkdownFile(file: File): boolean {
```

Comment:
Why do we need some specific for markdown?

Reply:
I assume it's a workaround for sending the correct mimetype if you have a .md file, as the browser would not do it by itself . But it looks like text/markdown is not in the list of supported types .. checking

Reply:
Actually we can't upload markdown , with or without this  (same in front). Removing the code as it's adding unnecessary complexity.